test/verifiedTests/analysis/testMultiSpeciesModelling/testMgPipe.m

```diff
@@ -134,3 +139,78 @@
 
 % change the directory
 cd(currentDir)
+
+% test getMappingInfo
+[reac,micRea,binOrg,patOrg,reacPat,reacNumb,reacSet,reacTab,reacAbun,reacNumber]=getMappingInfo(models,abunFilePath,indNumb);
+
+assert(exist('reac', 'var') == 1)
+assert(exist('micRea', 'var') == 1)
+assert(exist('binOrg', 'var') == 1)
+assert(exist('patOrg', 'var') == 1)
+assert(exist('reacPat', 'var') == 1)
+assert(exist('reacNumb', 'var') == 1)
+assert(exist('reacSet', 'var') == 1)
+assert(exist('reacTab', 'var') == 1)
+assert(exist('reacAbun', 'var') == 1)
+assert(exist('reacNumber', 'var') == 1)
+
+% test checkNomenConsist
+[autoStat,fixVec,organisms]=checkNomenConsist(organisms,autoFix);
+
+assert(exist('autoStat', 'var') == 1)
+assert(exist('fixVec', 'var') == 1)
+assert(length(organisms) == 5)
+
+% test fastSetupCreator
+
+setup=fastSetupCreator(models, organisms, {},objre);
+ 
+assert(exist('setup', 'var') == 1)
+assert(strcmp(class(setup),'struct') == 1)
+assert((length(setup.S) == length(setup.rxns)) == 1)
+assert((length(setup.lb) == length(setup.rxns)) == 1)  
+assert((length(setup.c) == length(setup.rxns)) == 1) 
+assert((length(setup.mets) == length(setup.metNames)) == 1)  
+ 
+assert(length(setup.rxns(strmatch('EX',setup.rxns))) /2 == length(setup.rxns(strmatch('DUt',setup.rxns))) == 1) 
+assert(length(setup.rxns(strmatch('EX',setup.rxns))) /2 == length(setup.rxns(strmatch('UFEt',setup.rxns))) == 1) 
+assert(length(setup.rxns(strmatch('DUt',setup.rxns)))== length(setup.rxns(strmatch('UFEt',setup.rxns))) == 1)
+
+assert((length(strmatch(organisms(1),setup.rxns)) > 0) == 1)
+assert((length(strmatch(organisms(2),setup.rxns)) > 0) == 1)
+assert((length(strmatch(organisms(3),setup.rxns)) > 0) == 1)
+assert((length(strmatch(organisms(4),setup.rxns)) > 0) == 1)
+assert((length(strmatch(organisms(5),setup.rxns)) > 0) == 1)
+
+% test createdModels
+[createdModels]=createPersonalizedModel(abunFilePath,resPath,setup,sampName,organisms,indNumb);
+assert(length(createdModels)== 5)
+assert(strcmp(createdModels(2,1),'Test1') == 1)
+assert(strcmp(createdModels(5,1),'Test4') == 1)
+
+microbiota_model=load(strcat(resPath,'microbiota_model_samp_Test1')); 
+microbiota_model=microbiota_model.microbiota_model;
+assert(length(microbiota_model.A)> length(microbiota_model.S))
```

Comment:
I would recommend not testing this (A/csense/S), as it is likely to become invalid in the future. 
In particular, as to the model definitions, length(csense) should be size(S,1). 

Also I would suggest to use an explicit size(S/A,1/2) to refer to the respective dimensions, as length can be either dimension and if someone else tries to figure out what a certain assert in a test means, this makes it really hard to detect potential problems.